SeleniumDemoQA/SeleniumDemoQA/Pages/FormPage.cs

```diff
@@ -15,16 +9,21 @@ internal class FormPage : BasePage
         private By firstNameInputBy = By.Id("firstName");
         private By lastNameInputBy = By.Id("lastName");
         private By emailInputBy = By.Id("userEmail");
+        private By genderRadioBy = By.CssSelector("label[for='gender-radio-1']");
         private By mobileNumberInputBy = By.Id("userNumber");
-        private By currentAddressInputBy = By.Id("currentAddress");
         private By dateOfBirthInputBy = By.Id("dateOfBirthInput");
         private By monthPickerBy = By.ClassName("react-datepicker__month-select");
         private By yearPickerBy = By.ClassName("react-datepicker__year-select");
+        private By datepickerDayBy = By.CssSelector(".react-datepicker__day--015:not(.react-datepicker__day--outside-month)");
         private By subjectsInputBy = By.Id("subjectsInput");
-        private By stateDropdownBy = By.Id("state");
-        private By cityDropdownBy = By.Id("city");
-        private By submitButtonBy = By.Id("submit");
-        private By confirmationModalBy = By.Id("example-modal-sizes-title-lg");
+        private By hobbiesCheckboxBy = By.CssSelector("label[for='hobbies-checkbox-1']");
+        private By currentAddressInputBy = By.Id("currentAddress");
+        private By stateBy = By.Id("state");
+        private By divTextNCRBy = By.XPath("//div[text()='NCR']");
```

Comment:
not clear name "divTextNCRBy"  and no point to have hardcoded selector By.XPath("//div[text()='NCR']");
It is better to have methods like
SetState(string state) and call it like somePage.SetState("NCR");
so inside this method you will have something like
_driver.ClickElement(stateBy);
 _driver.ClickElement(By.XPath($"//div[text()='{state}']"));

the same for divTextDelhiBy